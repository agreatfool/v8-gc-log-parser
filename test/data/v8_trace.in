[20671:0x103000000]      687 ms: Scavenge 11.5 (18.0) -> 7.7 (18.0) MB, 0.6 / 0.0 ms  allocation failure
[20869:0x103001000]      812 ms: Mark-sweep 11.0 (19.0) -> 6.8 (19.0) MB, 0.8 / 0.0 ms  (+ 5.8 ms in 9 steps since start of marking, biggest step 1.0 ms, walltime since start of marking 12 ms) finalize incremental marking via task GC in old space requested
[20671:0x103000000]      930 ms: Scavenge 10.9 (18.5) -> 7.2 (18.5) MB, 0.8 / 0.0 ms  allocation failure
