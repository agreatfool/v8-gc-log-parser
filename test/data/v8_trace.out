[
  {
    "pid": 20671,
    "isolate": "0x103000000",
    "time_since_init": 687,
    "data": {
      "collector": "scavenge",
      "start_object_size": 11.5,
      "end_object_size": 7.7,
      "start_memory_size": 18,
      "end_memory_size": 18,
      "pause": 0.6,
      "external_time": 0
    },
    "type": "trace"
  },
  {
    "pid": 20869,
    "isolate": "0x103001000",
    "time_since_init": 812,
    "data": {
      "collector": "mark_sweep",
      "start_object_size": 11,
      "end_object_size": 6.8,
      "start_memory_size": 19,
      "end_memory_size": 19,
      "pause": 0.8,
      "external_time": 0,
      "step_description": "+ 5.8 ms in 9 steps since start of marking, biggest step 1.0 ms, walltime since start of marking 12 ms"
    },
    "type": "trace"
  },
  {
    "pid": 20671,
    "isolate": "0x103000000",
    "time_since_init": 930,
    "data": {
      "collector": "scavenge",
      "start_object_size": 10.9,
      "end_object_size": 7.2,
      "start_memory_size": 18.5,
      "end_memory_size": 18.5,
      "pause": 0.8,
      "external_time": 0
    },
    "type": "trace"
  }
]
